client/components/dashboard/dashboard-model.js

```diff
@@ -133,6 +139,16 @@ explorer.components.dashboard.DashboardModel = function() {
    * @export
    */
   this.params = [];
+  
+  /**
+   * @type {{string}, {*}}
+   */
+  this.config = {
+    bigQuery: new BigqueryConfigModel()
+  };
+  
+  this.config.bigQuery.optimizeCurrentTimestamp.enabled = false;
```

Comment:
Moved bigQuery initialization outside of config initialization, and added a TODO to refactor them into a BigQuery-specific service.